src/components/Header/Header.js

```diff
@@ -6,7 +6,7 @@ const Header = ({ ChangePage, CurrentPage, Resume }) => {
     <nav className="navbar navbar-expand navbar-dark justify-content-between rezume-header">
       <span className="navbar-brand mb-0 h1 col-1">Rezume</span>
       <ul className="navbar-nav">
-        {Object.keys(Resume).map(NavItem => (
+        {Object.keys(Resume).map((NavItem) => (
```

Comment:
Remove parentheses.

Reply:
I didn't add them. The prettier added them. I copied the relevant configurations from the dotfile that you provided in the documentation. If you want I can remove it, but on every further save it will get added automatically. 

Here is the configuration that I copied from your dotfile, ( if any relevant config is missing let me know)
  "[html]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.wordWrap": "off"
  },
  "[javascriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
  "[javascript]": {
    "editor.formatOnSave": true,
    "editor.formatOnPaste": true
  },
  "files.associations": {
    "*.js": "javascriptreact",
    "*.inc": "php"
  },
  "[json]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },

if you still want those () to be removed let me know.

Reply:
@mayonk-kumar-git I know. I was talking about this, maybe this will help: https://gist.github.com/praveenscience/ebb5439f31774ad2fdc14cb9e7de1fc0

Reply:
@praveenscience I copied the configuration from this file only. I didn't copy the dotfile as a whole because I have my other preferences. But I copied the javascript-related configs and others that I mentioned above. Anyway, I am removing the (). 😄